<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NearestFeatureDialogBase</class>
 <widget class="QDialog" name="NearestFeatureDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nearest Feature</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditIp"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDbName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditUser"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPassword"/>
    </item>
    <item>
     <widget class="QLabel" name="labelSqlCmd">
      <property name="text">
       <string>Input Your Sql Cmd:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditSqlCmd"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditLayerName"/>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>400</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButtonGenerateLayer">
    <property name="text">
     <string>Generate Layer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTestConnect">
    <property name="text">
     <string>Test Connect</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="button_box">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>NearestFeatureDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>NearestFeatureDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
